components/first_time/first_time_component.js

```diff
@@ -1,126 +1,75 @@
-                                onPress={() => alert(
-                                    "Printing out input fields: \n" +
-                                    "Field 1: " + this.state.field1 + "\n" +
-                                    "Field 2: " + this.state.field2 + "\n" +
-                                    "Field 3: " + this.state.field3 + "\n" +
-                                    "Field 4: " + this.state.field4 + "\n" +
-                                    "Field 5: " + this.state.field5 + "\n"
-                                )}>
-                                <Text style={styles.contButtonText}>CONTINUE</Text>
-                            </TouchableOpacity>
-                        </View>
-                        <View style={{flex:7}} />
-                    </View>
-                    <View style={{flex:1, backgroundColor: "beige"}}/>
-                </View>
-            </ScrollView>
-        );
-    }
-};
-
-const styles = StyleSheet.create({
-    genericTextBox: {
-        backgroundColor: "white",
-        flex: 1,
-        height: 40,
-        borderColor: "gray",
-        borderWidth: 2,
-    },
-    contButton: {
-        backgroundColor: "blue",
-        flex: 1,
-        flexDirection: "row",
-        justifyContent: "center",
-        alignItems: "center"
-    },
-    contButtonText: {
-        fontSize: 24,
-        color:"white",
-        fontWeight:"bold"
-    }
-});
\ No newline at end of file
+  render () {
+    const { navigate } = this.props.navigation
+    return (
+      <Container>
+        <Header style={{backgroundColor: 'black'}}>
+          <Body>
+            <Title style={{fontWeight: 'bold', fontSize: 30}}>First-Time Setup</Title>
+          </Body>
+        </Header>
+        <Content padder>
+          <Form style={{backgroundColor: '#AAAAAA'}}>
+            <Item inlineLabel>
+              <Label>  Display Name: </Label>
+              <Input onChangeText={(input) => this.setState({field1: input})} />
+            </Item>
+            <Item inlineLabel>
+              <Label>  Date of Birth: </Label>
+              <Input onChangeText={(input) => this.setState({field2: input})} />
+            </Item>
+            <Item inlineLabel>
+              <Label>  Discord Tag: </Label>
+              <Input onChangeText={(input) => this.setState({field3: input})} />
+            </Item>
+            <Item inlineLabel>
+              <Label>  Misc Attribute1: </Label>
+              <Input onChangeText={(input) => this.setState({field4: input})} />
+            </Item>
+            <Item inlineLabel>
+              <Label>  Misc Attribute2: </Label>
+              <Input onChangeText={(input) => this.setState({field5: input})} />
+            </Item>
+          </Form>
+          <View style={{backgroundColor: '#999999', height: 15}} />
+          <View style={{height: 10}} />
+          <View>
+            <Button full primary
+              onPress={() => {
```

Comment:
You might consider moving this into a `handleSubmit` function, I think we've done that in a few other places around the project, and it makes it a bit more readable (at least in my eyes). @jpadgett7 might be able to speak on how he did his createRequest component.